.idea/codeStyles/Project.xml

```diff
@@ -1,5 +1,6 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
+    <option name="LINE_SEPARATOR" value="&#10;" />
```

Comment:
Do we need this setting for idea project?

Reply:
Removed.